docs/source/aws/usage.rst

```diff
@@ -35,10 +35,11 @@ flag needs to be specified when running Cloud Custodian::
 
 You can also consolidate metrics into a single account by specifying the ``master``
 location in the cli. Note that this is only applicable when using the ``--assume`` option
-in the cli or when using c7n-org. By default, metrics will be sent to the same account
-that is being executed against::
+in the cli or when using c7n-org. You can use ``profile`` query parameter to specify a
```

Comment:
setting profile this way feels a little odd, I'd actually prefer this in a separate pr, as the metrics functionality is otherwise good to go.

1. the way to specify profile is via standard sdk environment variables. ie. `AWS_PROFILE`
2. we typically have executing credentials for main/master directly from cli execution environment, and then target account credentials from --assume cli flag. this is introducing a third specific to metrics output, and its not clear what the use case is.
3. just noting we should probably be dropping user visible `master` references in favor of something else, although not sure what's the appropriate name here `main` feels odd.

Reply:
could you clarify the use case for profile here?

Reply:
I've got your point. Here are two scenarios: one specific to my situation and a more universal one.

In my case:
I utilize AWS SSO profiles configured in my local environment, which are set up in the ~/.aws/config file, to access AWS accounts. In my c7n_org accounts.yml file, I make use of the "profile" keyword. Consequently, when I use `aws://master`, it sends metrics data to the target account, because the `profile` value in accounts.yml asks it to. Introducing a `profile` query parameter would allow me to route the metrics data to the specific "master" account I desire.

In a more general context:
There may be instances when a user wishes to store metrics data in an account that differs from the designated `master` account. For instance, in the case of c7n_org is configured to utilize the credentials of the root account to access all target accounts, the user might want to store metrics data in a separate log account, instead of the root account.

Re #3, personally I don't mind what's the name as long as it help me to get my job done. :). In fact, I think `master` is a good one.

(I'll make the code changes tomorrow. It's been a bit hectic lately.)